src/main/java/racingcar/controller/GameControllerImpl.java

```diff
@@ -0,0 +1,71 @@
+package racingcar.controller;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.ArrayList;
+import java.util.List;
+import racingcar.util.style.DistanceStyle;
+import racingcar.util.style.DistanceStyleFactory;
+import racingcar.util.style.DistanceStyles;
+import racingcar.view.car.CarView;
+import racingcar.view.input.InputView;
+import racingcar.view.output.OutputView;
+import racingcar.vo.CarList;
+import racingcar.vo.CarListFactory;
+
+public class GameControllerImpl implements GameController {
+    private final OutputView outputView;
+    private final InputView inputView;
+    private final List<CarView> carViewList;
+
+    public GameControllerImpl(final OutputView outputView, final InputView inputView) {
+        this.outputView = outputView;
+        this.inputView = inputView;
+
+        carViewList = new ArrayList<>();
+    }
+
+    @Override
+    public void run() {
+        final CarList carList = getCarList();
+        createCarViews(carList);
+        final int tryCount = getTryCount();
+        outputView.printNewLine();
+        printExecutionResult(tryCount, carList);
+        printWinners(carList);
+        Console.close();
+    }
```

Comment:
```suggestion
    public void run() {
        final CarList carList;
        final int tryCount;
        initializeGame(carList, tryCount);
        // outputView.printNewLine(); -> printExcutionResult()로 이동
        printExecutionResult(tryCount, carList);
        printWinners(carList);
        // Console.close(); -> Application.main()으로 이동
    }
```
제 PR에서 코드의 구조를 바꾼 김에, 선재님 PR에서도 유사한 고민을 하시는 것 같아 바꿔보았습니다.
@wooteco-daram @cosyflower